pkg/cmd/workspace/stop.go

```diff
@@ -72,4 +78,27 @@ func init() {
 	}
 
 	StopCmd.Flags().StringVarP(&stopProjectFlag, "project", "p", "", "Stop a single project in the workspace (project name)")
+	StopCmd.Flags().BoolVarP(&allFlag, "all", "a", false, "Stop all workspaces")
+}
+
+func stopAllWorkspaces() {
```

Comment:
Same comment as with `deleteAllWorkspaces`. Return an error and handle it above.